pkgs/unified_analytics/lib/src/session.dart

```diff
@@ -118,6 +118,21 @@ class Session {
         description: err.osError?.toString(),
       ));
 
+      // Fallback to setting the session id as the current time
+      _sessionId = now.millisecondsSinceEpoch;
+      // ignore: avoid_catching_errors
+    } on TypeError catch (err) {
```

Comment:
let's not catch TypeError, and instead be disciplined about never deleting fields from this JSON file.